servicetalk-buffer-api/build.gradle

```diff
@@ -28,5 +28,12 @@ dependencies {
   testImplementation "org.junit.jupiter:junit-jupiter-params"
   testImplementation "org.hamcrest:hamcrest:$hamcrestVersion"
   testImplementation "org.mockito:mockito-core:$mockitoCoreVersion"
+
   testFixturesImplementation "org.hamcrest:hamcrest:$hamcrestVersion"
+
+  if (JavaVersion.current().isJava9Compatible()) {
+    compileTestFixturesJava {
+      options.release = compileJava.options.release
```

Comment:
Instead of adding this code snippet in every `build.gradle`, we should rely on our gradle plugin to set it. The plugin is a single source of truth for consistent gradle configuration across all modules.
This can be repeated for submodules that do not apply our plugin.

Reply:
This snippet is only in the `build.gradle` of modules which add `testFixtures`. Unless the test fixtures are added then the `compileTestFixturesJava` task is undefined which means this snippet can't be added to the plugin. I was unable, thus far, to find a way to make the behaviour conditional so that happened at the right time and only for the projects with a `compileTestFixturesJava` task.

Reply:
Gradle allows searching for tasks by their string name: `tasks.findByName("someName")`. It helps to apply logic conditionally. There are a few examples inside our plugin.